src/Profiles.ts

```diff
@@ -601,208 +426,36 @@ export class Profiles {
-                        "Operation Cancelled"));
-                    return undefined;
-                }
-                schemaValues[value] = newPass;
-                break;
-            case "rejectUnauthorized" :
-                newRU = await this.ruInfo();
-                if (newRU === undefined) {
-                    vscode.window.showInformationMessage(localize("createNewConnection.rejectUnauthorize",
-                    "Operation Cancelled"));
-                    return undefined;
-                }
-                schemaValues[value] = newRU;
-                break;
-            default:
-                let options: vscode.InputBoxOptions;
-                const response = await this.checkType(schema[value].type);
-                switch (response) {
-                    case "number" :
-                        options = await this.optionsValue(value, schema);
-                        const enteredValue = await vscode.window.showInputBox(options);
-                        if (!Number.isNaN(Number(enteredValue))) {
-                            schemaValues[value] = Number(enteredValue);
-                            } else {
-                                if (schema[value].optionDefinition.hasOwnProperty("defaultValue")){
-                                    schemaValues[value] = schema[value].optionDefinition.defaultValue;
-                                } else {
-                                    schemaValues[value] = undefined;
-                                }
-                            }
-                        break;
-                    case "boolean" :
-                        let isTrue: boolean;
-                        isTrue = await this.boolInfo(value, schema);
-                        if (isTrue === undefined) {
-                            vscode.window.showInformationMessage(localize("createNewConnection.booleanValue",
-                            "Operation Cancelled"));
-                            return undefined;
-                        }
-                        schemaValues[value] = isTrue;
-                        break;
-                    default :
-                        options = await this.optionsValue(value, schema);
-                        const defValue = await vscode.window.showInputBox(options);
-                        if (defValue === "") {
-                            break;
-                        }
-                        schemaValues[value] = defValue;
-                        break;
-                }
-            }
-        }
-
         try {
+            const newProfileDetails = await ZoweExplorerApiRegister.getCommonApi(profileLoaded)
+                                                                   .collectProfileDetails(null,
+                                                                                          profileLoaded.profile,
+                                                                                          await this.getSchema("zosmf"));
```

Comment:
you cant hard code "zosmf" here for the creating of profiles we need the check that was there for the different profile types and that getCommonApi() commented above is a breaking change for extenders

Reply:
Fixed this one